index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Survey Form</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+<body>
+    <div id="head">
+        <h1 id=title>freeCodeCamp Survey Form</h1>
+        <p id="desc"><i>you for taking the time to help us improve the platform</i></p>
+    </div>
+    <div id="main">
+        <form>
+            <div>
+                <label><b>Name:</b></label>
+                <input type="name" name="name" class="form" required placeholder="Full Name"><br>
+                <br>
+                <label><b>Email:</b></label>
+                <input type="email" name="email" class="form" required placeholder="example@mail.com"><br>
+                <br>
+                <label><b>Age:</b></label>
+                <input type="number" name="number" class="form" min="5" max="99" placeholder="12+">
+            </div>
```

Comment:
This looks amazing✨ I think it would be better to use semantic tags. we could use `section` instead of `div` for better SEO optimization and clean codes.